lp_bookdown/Demographics.Rmd

```diff
@@ -0,0 +1,132 @@
+
+\newpage
+
+# Demographics
+
+\newline
```

Comment:
I assume `\newpage` is creating a page break?

I don't think we need `\newline` though. I could be wrong as I know it appears a lot generally 😆